extensions/infinispan-client/deployment/src/main/java/io/quarkus/infinispan/client/deployment/devservices/InfinispanDevServicesConfig.java

```diff
@@ -74,6 +74,31 @@ public class InfinispanDevServicesConfig {
     @ConfigItem
     public Optional<List<String>> artifacts;
 
+    /**
+     * When creating a Infinispan container with cross site replication
```

Comment:
Is something missing in the comment? It seems to end abruptly and not really explain what the property sets

Reply:
I changed the explanation

Reply:
Are you sure? It seems the same to me :)